src/magellan_gameplay/package.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>magellan_gameplay</name>
+  <version>0.0.0</version>
+  <description>The magellan_gameplay package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="magellan@todo.todo">magellan</maintainer>
```

Comment:
Unimportant, but we have ras@pitt.edu on other packages (similar to IARC)

Reply:
fixed